src/app/modules/email-modal/email-modal.component.ts

```diff
@@ -237,11 +240,17 @@ export class EmailModalComponent implements OnInit, OnDestroy, AfterContentInit
                     duration: 2000,
                 });
                 this.commonService.inboxRefreshEvent();
+                broadcast_send({ 'any':'value' });
             }, (err) => {
                 console.log(err);
             });
             this._location.back();
         }
+
+        function broadcast_send(value) {
+            localStorage.setItem('changed', JSON.stringify(value));
```

Comment:
setItem and removeItem in next line?? please explain

Reply:
Ok, do 
localStorage.setItem('updateInbox', {} );
localStorage.removeItem('updateinbox) like this